### PR TITLE
Implemented accuracy

### PR DESCRIPTION
**Pull request by YashitaBansal**
Inputs:
`std::vector<int> pred = {1, 0, 1, 1, 0, 1};
  std::vector<int> trueLabels = {1, 0, 0, 1, 0, 1};`

Output:
Accuracy:0.8333

### CODE REVIEW

**Review by uttammittal02 — APPROVED** [review]
Nice work

File: src/slowmokit/methods/metrics/accuracy.cpp [hunk 1]
```diff
@@ -0,0 +1,25 @@
+/**
+ * @file methods/metrics/accuracy.cpp
+ *
+ * Implementation of the Accuracy score main program
+ */
+#include "accuracy.hpp"
+template <class T>
+double accuracy(const std::vector<T> &pred, const std::vector<T> &true_labels)
```
Comment (ken1000minus7) [inline src/slowmokit/methods/metrics/accuracy.cpp:8]: Variable names should be in camel case according to our conventions

**Review by ken1000minus7 — CHANGES_REQUESTED** [review]
Lgtm, just a small change needed

**Review by ken1000minus7 — APPROVED** [review]
Great work 👌

File: src/slowmokit/methods/metrics/accuracy.cpp (hunk 1 above)
Comment (YashitaBansal) [reply src/slowmokit/methods/metrics/accuracy.cpp]: Sir I have done the changes, is it visible to you or should I open pull request again?

**Comment by YashitaBansal** [comment]
> Nice work

Thank you sir

**Comment by YashitaBansal** [comment]
> Great work 👌

Thank you sir😇

**Pull request merged**